src/ts/components/subtitleoverlay.ts

```diff
@@ -85,6 +85,13 @@ export class SubtitleOverlay extends Container<SubtitleOverlayConfig> {
 
       this.applyConfToDom(labelToAdd.getDomElement());
 
+      if (event.position != null) {
+        let domElement = labelToAdd.getDomElement();
+        domElement.css('position', 'fixed');
+        domElement.css('left', `${event.position.column}em`);
```

Comment:
So that basically means that CEA-608 is incompatible to the FCC requirements. To solve the font problem, we could create a grid and render letters separately into it... it'd work with any font, but you couldn't control the font size any more. 

Maybe we shouldn't allow FCC settings on CEA-608 captions, or limit the settings to those that do not interfere (e.g. color settings)? We could also apply a monospaced default profile to the FCC settings to get correct rendering. I'm not really happy with the current solution (not your fault!) and I'm going to open an issue about potential improvements.